front/src/applications/stdcm/utils/checkStdcmConfigErrors.ts

```diff
@@ -8,14 +10,42 @@ import { StdcmConfigErrorTypes, ArrivalTimeTypes, type StdcmConfigErrors } from
 const checkStdcmConfigErrors = (
   pathSteps: StdcmPathStep[],
   t: TFunction,
-  pathfindingStatus?: 'success' | 'failure'
+  pathfindingStatus?: 'success' | 'failure',
+  rollingStock?: RollingStockWithLiveries,
+  totalMass?: number | undefined,
+  totalLength?: number | undefined,
+  maxSpeed?: number | undefined,
+  isSimulationLaunched?: boolean
```

Comment:
This way it's more obvious when we use the function
```suggestion
  options?: { isSimulationLaunched: boolean }
```

Reply:
done